### PR TITLE
multi-references-for-protobuf

### PR DESCRIPTION
**Pull request by bifrost**

### CODE REVIEW

**Review by mickael-h — APPROVED** [review]
Awesome work!

**Review by msafwat-sumup — APPROVED** [review]

File: src/JsonHelper.ts [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+  SchemaType,
+  ReferenceType,
+  JsonConfluentSchema,
+  ProtocolOptions,
+} from './@types'
 import { ConfluentSchemaRegistryError } from './errors'
 
 export default class JsonHelper implements SchemaHelper {
-  public validate(schema: Schema): void {
+  public validate(_schema: Schema): void {
```
Comment (soulmonk) [inline src/JsonHelper.ts:15]: a little confusion with naming, in the interface `SchemaHelper` without underscore `validate(schema: Schema): void` - not critical but better to have consistency.

File: src/JsonHelper.ts [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
     return
   }
 
   public getSubject(
-    confluentSchema: ConfluentSchema,
-    schema: Schema,
-    separator: string,
+    _confluentSchema: ConfluentSchema,
+    _schema: Schema,
+    _separator: string,
```
Comment (soulmonk) [inline src/JsonHelper.ts:22]: same here, consistency with `SchemaHelper`

Comment (bifrost) [reply src/JsonHelper.ts:22]: I agree - but this is how TS was set up. It marks it as a problem if unused, to fix it add underscore.

Comment (soulmonk) [reply src/JsonHelper.ts:22]: then I think better to use: `@ts-ignore unused variable` for this line or something like this ?
and comment remove after implementation.

**Pull request merged**